vignettes/cookbook.Rmd

```diff
@@ -112,12 +115,19 @@ The scenario data set provides information on the trajectories of technologies/f
 For sectors with technology level trajectories, the data set provides the TMSR and SMSP pathways based on the Market Share Approach, an allocation rule that implies all companies active in a sector have to adjust their production in a way that keeps market shares constant and solves for the aggregate climate transition scenario (LINK to the market share documentation).
 
 The target market share scenario data set must be a CSV file and contains the following columns:
-**ADD**
+```{r cols_tms_scenario, echo = FALSE}
+# TODO: should this be in the data dictionary?
+cat(paste(names(pacta.multi.loanbook:::col_types_scenario_tms[["cols"]]), collapse = ",\n"))
+```
+
 
 For sectors that do not have technology level pathways, PACTA uses the Sectoral Decarbonization Approach (SDA), an allocation rule that implies that all companies in a sector have to converge their physical emission intensity at a future scenario value - e.g. in the year 2050. This implies that more polluting companies have to reduce their physical emissions intensity more drastically than companies using cleaner technology. It does not have any direct implications on the amount of units produced by any company (LINK to the SDA documentation).
 
 The SDA scenario data set must be a CSV file and contains the following columns:
-**ADD**
+```{r cols_sda_scenario, echo = FALSE}
+# TODO: should this be in the data dictionary?
+cat(paste(names(pacta.multi.loanbook:::col_types_scenario_sda[["cols"]]), collapse = ",\n"))
+```
```

Comment:
```suggestion

```{r cols_sda_scenario, echo = FALSE, results = 'asis'}
# TODO: should this be in the data dictionary?
cat(paste0("- `", names(pacta.multi.loanbook:::col_types_scenario_sda[["cols"]]), "`", collapse = "\n"))
```
```